### PR TITLE
Added possibility to treat some NVRS as global.

### PR DESCRIPTION
**Pull request by judovana**

### CODE REVIEW

File: koji-scm-lib/src/main/java/hudson/plugins/scm/koji/NotProcessedNvrPredicate.java [hunk 1]
```diff
@@ -41,23 +41,35 @@
...
 
-    public static Predicate<String> createNotProcessedNvrPredicateFromFile(File processedNvrFile) throws IOException {
-        if (processedNvrFile.exists()) {
+    public static Predicate<String> createNotProcessedNvrPredicateFromFile(File processedNvrFile, File globalprocessedNvrFile) throws IOException {
+        List<String> l1 = fileToList(processedNvrFile);
+        List<String> l2 = fileToList(globalprocessedNvrFile);
+        List<String> l3 = new ArrayList<>(l1.size()+l2.size());
+        l3.addAll(l1);
+        l3.addAll(l2);
+        return new NotProcessedNvrPredicate(l3);
```
Comment (oklinov) [inline koji-scm-lib/src/main/java/hudson/plugins/scm/koji/NotProcessedNvrPredicate.java]: These lists could be named better :)

Comment (judovana) [reply koji-scm-lib/src/main/java/hudson/plugins/scm/koji/NotProcessedNvrPredicate.java]: right you are

Comment (judovana) [reply koji-scm-lib/src/main/java/hudson/plugins/scm/koji/NotProcessedNvrPredicate.java]: fixed

**Comment by judovana** [comment]
How about backport to 2?

**Pull request merged**